guides/incremental-delivery.md

```diff
@@ -0,0 +1,483 @@
+# Incremental Delivery
```

Comment:
I see this, `INCREMENTAL_DELIVERY.md` and `README_INCREMENTAL.md` and they look similar. Could one or more be removed and consolidated?